src/internal.c

```diff
@@ -17767,11 +17787,6 @@ int SendCertificateVerify(WOLFSSL* ssl)
                                                                 args->inputSz);
             }
 
-            /* Check for error */
-            if (ret != 0) {
```

Comment:
ret isn't set anyways between the previous one and this one. So its dead code, although if someone did add a "ret = " in that section they would need to make sure and check/handle it.